src/main/java/org/hmhb/authentication/DefaultJwtAuthenticationService.java

```diff
@@ -126,85 +132,97 @@ public void validateAuthentication(
 
             String authToken = authHeaderInfo[1];
 
-            if (!Jwts.parser().isSigned(authToken)) {
```

Comment:
The diff on this isn't amazing.  Basically, I took the inner piece of the validateAuthentication method (used by the AuthenticationFilter to check the Authorization header) and exposed it (so the JWT token could be validated by itself).